eleventy.config.js

```diff
@@ -165,6 +166,72 @@ export default async function (eleventyConfig) {
 	eleventyConfig.amendLibrary("md", (mdLib) => {
 		mdLib.use(markdownItFootnote);
 
+		// Add container support for note types
+		const noteTypes = ["note", "warning", "success", "info"];
+		const noteTypePattern = new RegExp(`^\\[!(${noteTypes.join("|")})\\]\\s*`, "i");
+
+		noteTypes.forEach((type) => {
+			mdLib.use(markdownItContainer, type, {
+				render: function (tokens, idx) {
+					if (tokens[idx].nesting === 1) {
+						// opening tag
+						return `<pob-note type="${type}">\n`;
+					} else {
+						// closing tag
+						return "</pob-note>\n";
+					}
+				},
+			});
+		});
```

Comment:
I don't like using `forEach`. Please convert this to a standard `for` loop. Do this for other locations, if you're using `forEach`.